executing/executing.py

```diff
@@ -121,27 +135,32 @@ def _get_instructions(co):
             yield Instruction(offset, argval, opname[op], lineno)
 
 
-try:
-    function_node_types += (ast.AsyncFunctionDef,)
-except AttributeError:
-    pass
+# Type class used to expand out the definition of AST to include fields added by this library
+# It's not actually used for anything other than type checking though!
+class EnhancedInstruction(Instruction):
+    _copied = None # type: bool
+
 
 
 def assert_(condition, message=""):
+    # type: (bool, str) -> None
     """
     Like an assert statement, but unaffected by -O
     :param condition: value that is expected to be truthy
     :type message: Any
     """
+    assert condition, message
```

Comment:
Why?

Reply:
I think because there was a case at some point where the `assert_` was called, but it didn't seem to fix the type limits, so the explicit `assert` was understood by mypy. This appears to no longer be the case, and have removed it.